tests/lean/server/diags.lean

```diff
@@ -1,5 +1,8 @@
 import Lean.Data.Lsp
 open IO Lean Lsp
+open Lean
+open Lean.Lsp
+open Lean.JsonRpc
```

Comment:
?

Reply:
yeah, it was a left over from before Ipc.shutdown refactor.  Fixed.